internal/ue/ue.go

```diff
@@ -67,19 +67,20 @@ type PDUSessionInfo struct {
 }
 
 type UE struct {
-	UeSecurity             *UESecurity
-	StateMM                int
-	DNN                    string
-	PDUSessionID           uint8
-	Snssai                 models.Snssai
-	amfInfo                Amf
-	IMEISV                 string
-	Gnb                    air.UplinkSender
-	mu                     sync.Mutex
-	PDUSession             PDUSessionInfo
-	receivedNASGMMMessages map[uint8][]*nas.Message // msgType -> gmm messages
-	receivedNASGSMMessages map[uint8][]*nas.Message // msgType -> gsm messages
-	receivedRRCRelease     bool
+	UeSecurity                   *UESecurity
+	StateMM                      int
+	DNN                          string
+	PDUSessionID                 uint8
+	Snssai                       models.Snssai
+	amfInfo                      Amf
+	IMEISV                       string
+	Gnb                          air.UplinkSender
+	mu                           sync.Mutex
+	PDUSession                   PDUSessionInfo
+	receivedNASGMMMessages       map[uint8][]*nas.Message // msgType -> gmm messages
+	receivedNASGSMMessages       map[uint8][]*nas.Message // msgType -> gsm messages
+	receivedRRCRelease           bool
+	configurationUpdateListeners []chan bool
```

Comment:
I don't think we need this channel with what we already have. Please see my other comment below.